hugolib/site_render.go

```diff
@@ -130,6 +130,10 @@ func (s *Site) renderPaginator(p *Page) error {
 
 func (s *Site) renderRSS(p *Page) error {
 
+	if !s.isEnabled(kindRSS) {
+		return nil
+	}
+
 	if s.Cfg.GetBool("disableRSS") {
```

Comment:
*Hmm...didn't save the comment I meant to make here...*

Can the enable/disable options update `disableKinds` in the `command` pkg so we can remove these checks?